Octokit.Tests.Integration/Clients/RepositoryCommentsClientTests.cs

```diff
@@ -310,12 +310,13 @@ public class TheCreateMethod : IDisposable
     {
         private readonly IGitHubClient _github;
         private readonly RepositoryContext _context;
+        private readonly IReactionsClient _reactionsClient;
```

Comment:
From a consistency perspective, can we not declare this class level subclient, and just use `_github.Reaction.Create()` etc where requried?  Simply because we dont have class levels for any other subclients in the test so we shouldnt add one now